config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "GITHUB_TOKEN": "02c488529ccb97674c4655467617f0442f37cee6",
```

Comment:
typically setting [environment variables](https://docs.travis-ci.com/user/environment-variables/) is the recommended approach to avoid including sensitive content in version control.

given that the site is going to be deployed live (using gh-pages), we might need to leverage a bit of node middleware in order to effectively hide a github token.